modules/layers/src/scatterplot-layer/scatterplot-layer.ts

```diff
@@ -103,8 +103,7 @@ export default class ScatterplotLayer<DataT = any, ExtraPropsT = {}> extends Lay
   }
 
   initializeState() {
-    // @ts-ignore (TS2531) attributeManager is always defined for primitive layer
-    this.getAttributeManager().addInstanced({
+    this.getAttributeManager()!.addInstanced({
```

Comment:
Should we really allow AttributeManager to be undefined? Can't we prototype getAttributeManager to always return  a valid instance, and do checks during initialization instead?